client/src/actions/index.js

```diff
@@ -1,148 +1,151 @@
-        history.push('/signin');
-      })
-      .catch(() => {
-        dispatch(authError('Failed to register user'));
-      });
-  };
-};
-
-// updateUser needs work
-export const updateUser = (username, password, confirmPassword, newPassword, confirmNewPassword, history) => {
-  return dispatch => {
-    if (password !== confirmPassword) {
-      dispatch(authError('Old passwords do not match'));
-      return;
-    }
-    if (newPassword !== confirmNewPassword) {
-      dispatch(authError('New passwords do not match'));
-      return;
-    }
-    axios
-      .post(`${ROOT_URL}/auth/reset`, { username, password, confirmPassword, newPassword, confirmNewPassword })
-      .then(() => {
-        dispatch({
-          type: USER_REGISTERED
-        });
-        history.push('/cards');
-      })
-      .catch(() => {
-        dispatch(authError('Failed to update user password'));
-      });
-  };
-};
-export const downloadCards = () => {
-  return dispatch => {
-    window.location.href = `${ROOT_URL}/cards/download`;
-  }
-}
-
-export const login = (username, password, history) => {
-  return dispatch => {
-    axios
-      .post(`${ROOT_URL}/auth/login`, { username, password })
-      .then((res) => {
-        dispatch({
-          type: USER_AUTHENTICATED
-        });
-        localStorage.setItem('token', res.data.token);
-        localStorage.setItem('user', JSON.stringify(res.data.user));
-        console.log(res.data);
-        history.push('/cards');
-      })
-      .catch(() => {
-        dispatch(authError('Incorrect email/password combo'));
-      });
-  };
-};
-
-export const logout = () => {
-  return dispatch => {
-    localStorage.removeItem('user');
-    localStorage.removeItem('token');
-    dispatch({
-      type: USER_UNAUTHENTICATED
-    });
-  };
-};
-
-export const getCards = () => {
-  return dispatch => {
-    axios
-      .get(`${ROOT_URL}/cards`)
-      .then((res) => {
-        dispatch({
-          type: GET_CARDS,
-          payload: res
-        });
-      })
-      .catch(() => {
-        dispatch(authError('Failed to get cards'));
-      });
-  };
-};
-
-export const getCard = (id) => {
-  return dispatch => {
-    axios
-      .get(`${ROOT_URL}/cards/${id}`)
-      .then((res) => {
-        dispatch({
-          type: GET_CARD,
-          payload: res
-        });
-      })
-      .catch(() => {
-        dispatch(authError('Failed to get card by that id'));
-      });
-  };
-};
-
-export const addCard = (card) => {
-  return dispatch => {
-    axios
-      .post(`${ROOT_URL}/card/create`, card)
-      .then((res) => {
-        dispatch({
-          type: ADD_CARD,
-          payload: res
-        });
-      })
-      .catch(() => {
-        dispatch(authError('Failed to add new card'));
-      });
-  };
+/* eslint-disable */
+import axios from 'axios';
+axios.defaults.withCredentials = true;
+const ROOT_URL = 'http://localhost:8080';

```

Comment:
This needs to be 
`-const ROOT_URL = process.env.NODE_ENV === 'production' ? 'https://bangarangbingo.herokuapp.com' : 'http://localhost:8080';`

or you will break the heroku build.  Please do daily `git pulls` to avoid this discrepency.